Documentation/release-notes/5051.md

```diff
@@ -0,0 +1,7 @@
+#### Application and library build and deployment
+
+  * [Developer Community 1157593][0]: Fixes an issue where modifying
+    `@(AndroidAsset)` files would require a `Rebuild` to see the
+    changes reflected in the Android application build output.
+
+[0]: https://developercommunity.visualstudio.com/content/problem/1157593/apk-is-not-rebuild-when-androidasset-changes.html
```

Comment:
Release note approved. Thanks!

In the final collated release notes, I'll plan to adjust the note slightly to mimic the past tense from the commit message and to use IDE terminology "Build Action" for `AndroidAsset`. Feel free to comment back if anything looks bad about this adjustment though.

```markdown
#### Application and library build and deployment

  * [Developer Community 1157593][0]: Modifications to files set to use
    the **AndroidAsset** Build Action were not included as expected in
    incremental builds. Clean rebuilds were required to propagate the
    changes as expected.

[0]: https://developercommunity.visualstudio.com/content/problem/1157593/apk-is-not-rebuild-when-androidasset-changes.html
```